core/src/test/java/nl/nn/adapterframework/xslt/XsltErrorTestBase.java

```diff
@@ -179,7 +179,7 @@ public void documentIncludedInSourceNotFoundXslt2() throws Exception {
 		try {
 			doPipe(pipe, input, session);
 		} catch (AssumptionViolatedException e) {
-			assumeTrue("assumption violated:"+e.getMessage(),false);
+			assumeTrue(false,"assumption violated:"+e.getMessage());
```

Comment:
Eigenlijk zijn al deze `assumeTrue(false)` een beetje vreemd.. wat moet hij nu precies testen? Kun je deze checken en corrigeren? Als ik er zo naar kijk snap ik iig niet de bedoeling. Heb liever dat er gewoon een `assertTrue` of `assertFalse` staat bv.

Reply:
Is deze assertion niet gewoon overbodig?

Reply:
Ja, daar lijkt het wel op. Die mag je dus opruimen.

Reply:
Mijn vorige reactie was niet helemaal juist. Door deze vage constructie gooit hij nog wel een exception.. Maar normaal los je dat niet zo op. Als hij iets gooit, terwijl dat niet moet, moeten we die juist doorgooien bv.